packages/graphql-auth-transformer/tsconfig.json

```diff
@@ -1,11 +1,10 @@
 {
     "compilerOptions": {
-        "target": "es5",
+        "target": "es6",
```

Comment:
Why are you making this change? I am not saying it won't work but we have customers that run old versions of node locally and some es6 features may not be supported in those environments. Unless we have a compelling reason lets not make these changes without really thinking through it.

Reply:
We're transpiling to ES5 with Babel and the new ```cloudform``` will throw: ```TypeError: Class constructor ResourceBase cannot be invoked without 'new'``` errors. Since Node 8 - which is the minimum version to run the CLI - supports ES6 natively and it would simplify the CLI code by removing unneeded polyfills from the released code.